docker/install.sls

```diff
@@ -3,6 +3,8 @@
 {%- set docker_pkg_name = docker.pkg.old_name if docker.use_old_repo else docker.pkg.name %}
 {%- set docker_pkg_version = docker.version | default(docker.pkg.version) %}
 {%- set docker_packages = docker.kernel.pkgs + docker.pkgs %}
+{%- set python_major = grains['pythonversion'][0] | string %}
```

Comment:
Just checking one thing - I have never used that grain value - is it always populated? 
`{%- set python_major = 3 if not grains['pythonversion'][0] else ['pythonversion'][0]|string %}`
`{%- set python_minor = `7` if not grains grains['pythonversion'][1] else grains['pythonversion'][1] | string %` %}`
And being really paranoid is it always defined?

Reply:
Just a cross-reference to another PR dealing with the same issue right now:

* https://github.com/saltstack-formulas/libvirt-formula/pull/47/files#diff-5ec9631f5880e09dff9a91b7dde192c3

Reply:
@greut squash the commits anyway. I'm not saying anything wrong with implementation - we can assume all (most) OS has this grain and merge this if you have tested.